src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
 }
+
+header {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  text-transform: uppercase;
+  width: 100%;
+  padding-inline: 50px;
+  box-sizing: border-box;
+}
+
+.is-active {
+  display: flex;
+  align-items: center;
+  color: #00acdc;
+  position: relative;
+  margin-left: auto;
+  height: 60px;
+  text-decoration: none;
+}
+
+.logo {
+  display: flex;
+}
+
+ul {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_link {
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class name `.nav_link` seems to be a typo and should be `.nav__link` according to BEM methodology.